packages/aws-cdk-lib/aws-ec2/test/instance.test.ts

```diff
@@ -904,4 +904,73 @@ test('specify ebs optimized instance', () => {
     InstanceType: 't3.large',
     EbsOptimized: true,
   });
-});
\ No newline at end of file
+});
+
+test.each([
+  [true, true],
+  [false, false],
+])('given nitroEnclaveEnabled %p', (given: boolean, expected: boolean) => {
```

Comment:
it needs to change the sentence If we change the parameter name.

Reply:
Updated to match the parameter name.